cli/status/lua/instance_state.lua

```diff
@@ -0,0 +1,57 @@
+local ok, config = pcall(require, 'config')
+if ok and config ~= nil and type(config) == "table" and type(config.info) == "function" then
+    config = config:info()
+else
+    config = {
+        status = "uninitialized",
+        alerts = {},
+    }
+end
+
+local config_info = {
+    status = config.status,
+    alerts = {}
+}
+for _, alert in pairs(config.alerts) do
+    table.insert(config_info.alerts, {
+        type = alert.type,
+        message = alert.message
+    })
+end
+
+local ok, box_replication = pcall(function() return box.info.replication end)
+if not ok or box_replication == nil then
+    box_replication = {}
+end
+
+local replication_info = {}
+for _, instance in ipairs(box_replication) do
+    table.insert(replication_info, {
+        name = instance.name,
+        upstream = instance.upstream and {
+            status = instance.upstream.status or "--",
+            message = instance.upstream.message
+        } or nil
+    })
+end
```

Comment:
We skip items without the `upstream` field on the Go side, so, maybe, don't add them into the result at all?

```lua
for _, instance in ipairs(box_replication) do
    if instance.upstream ~= nil then
        table.insert(<...>, <...>)
    end
end